Doc/whatsnew/3.11.rst

```diff
@@ -176,14 +176,33 @@ The :option:`-X` ``no_debug_ranges`` option and the environment variable
 See :pep:`657` for more details. (Contributed by Pablo Galindo, Batuhan Taskaya
 and Ammar Askar in :issue:`43950`.)
 
+Exception Groups and ``except*`` (PEP 654)
```

Comment:
```suggestion
and Ammar Askar in :issue:`43950`.)


Exception Groups and ``except*`` (PEP 654)
```

It isn't consistent, but generally two blank lines are currently used before h3 headings in the current document, and in particular that may help reduce merge conflicts between PRs that modify adjacent sections (e.g. I have #95915 open right now), so it's probably best to keep to that